docs/SCP Roguelike.md

```diff
@@ -1,203 +1,110 @@
+* Containment breach -- identical to the eponymous game mode
+* Area attack -- attack on a FOB of a rival world conspiracy: blast through the defences, get to the control room and geek the local boss.
+* Area defence -- do not let rival world conspiracy agents to do the same on you
+* Area exploration -- explore the pocket dimension/internals of a giant SCP/the other side of a portal
+* Object containment -- secure and contain (or destroy) an actively opposing anomalous entity
+* Raid -- ~~steal~~ acquire an object of interest from a rival world conspiracy
 
-Условия содержания для каждого объекта генерируются случайным образом, так же, как и свойства каждого объекта (хотя можно и нужно делать какую-то логическую связь между условиями и свойствами объектов). Для некоторых объектов вместо восстановления условий должно быть допустимо их устранение, но это должно быть непросто сделать (особенно по отношению к объектам класса Кетер).
+# Factions
 
-Нарушение условий содержания одного из объектов может нарушить условия и для других объектов, что создаёт условия для локального события класса TCF и ещё большего веселья.
+Factions should be described on the following terms:
+* Equipment access: guns, armor, tanks, orbital killsats, etc
+* Equipment quality: A rusty AK is better than nothing. But there are better alternatives.
+* Anomalous objects access: how many objects organization has and how willing it is to use them
+* Combat ability: how good organization members are at dishing out harm and surviving
+* Special traits
 
-При нарушении условий содержания должна учитываться инфраструктура Зоны (как минимум - наличие и расположение вентиляции и электрической сети).
 
-## Захват объектов
+## SCP Foundations
+Focuses on accessible equipment and research.
 
-Один или несколько агентов Фонда в полевых условиях осуществляют захват опасного объекта. Возможные вариации:
+* Equipment access: high
+* Equipment quality: average
+* Anomalous objects access: low
+* Combat ability: average
 
-* захват "социального" объекта в городских условиях с участием гражданского населения (объекты могут захватывать гражданских и т.п.)
+### Special traits
+* Body of knowledge
+* Institutional paranoia
 
-* отобрать объект у связанной организации;
+## Chaos Insurgency
+Focuses on anomalous object usage
+* Equipment access: average
+* Equipment quality: average
+* Anomalous objects access: high
+* Combat ability: low
 
-* отобрать объект у Фонда, будучи на стороне связанной организации;
+### Special traits
+* No holds barred
 
-* отобрать объект у Фонда, будучи на стороне Фонда (двойной агент и т.п.);
+## Global Occult Coalition
+Focuses on equipment quality
+* Equipment access: average
+* Equipment quality: high
+* Anomalous objects access: none
+* Combat ability: high
 
-* отобрать объект у монстров.
+### Special traits
+* Killswitch engage
+* The Last Stand
 
-## Нападение на Зону
+## The Factory
```

Comment:
On the contrary, there is a lot of objective evidence (relevant SCPs and first-hand accounts). Except the conclusions derived from this evidence is mostly wrong. So we pretty much know what The Factory does, but **how and why** are pretty much known unknowns.